src/components/Beacon.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Helmet } from 'react-helmet'
+
+const HelpScoutBeacon: React.FC = () => (
```

Comment:
Can you name the file the same name as the component? It helps to keep it consistent.